python/RooAddPdfFixer.py

```diff
@@ -10,18 +10,23 @@
 FixAll(w)
 """
 
+from __future__ import absolute_import
+
 import ROOT
 
 __inited = False
 
+
 def __init():
-  global __inited
-  if __inited: return
-  __inited = True
-  ROOT.gSystem.Load("libHiggsAnalysisCombinedLimit")
-  ROOT.gROOT.ProcessLine("#include <HiggsAnalysis/CombinedLimit/interface/utils.h>")
+    global __inited
+    if __inited:
+        return
+    __inited = True
+    ROOT.gSystem.Load("libHiggsAnalysisCombinedLimit")
+    ROOT.gROOT.ProcessLine("#include <HiggsAnalysis/CombinedLimit/interface/utils.h>")
+
 
 def FixAll(workspace):
-  __init()
-  fixer = ROOT.utils.RooAddPdfFixer()
-  fixer.FixAll(workspace)
```

Comment:
Given RooAddPdfFixer has been removed from `interface/utils.h` I think we should just delete this.

Reply:
Thanks. For reference to others: https://github.com/cms-analysis/HiggsAnalysis-CombinedLimit/commit/ecfc7eedd9fe3d506c7daf07be3fe664b68466e5#diff-e61e63e960ab2636e94f487a3632a681d4443cb9461dc61667a91fce743ab613L124-L137 removed it from c++ side.